src/pages/ContactUsView.jsx

```diff
@@ -67,6 +65,30 @@ export default function ContactUsView() {
     (id === 'message') ? parseAndApplyEmailInput(formData.subject.value, value) : parseAndApplyEmailInput(value, formData.message.value);
   }
 
+  /**
+   * If all required values aren't set then it will prevent link functionality
+   * Will set error message where required
+   * @param {Event} event
+   */
+  function handleSendClicked(event) {
+    //target for href is set so allow link functionality
+    if (mailTo) {
+      return;
+    }
```

Comment:
I have something against the empty return statement. I just don't know what!

Reply:
Welp :,(